src/server/plugins/register-form/steps/registered-name-details.js

```diff
@@ -9,7 +9,7 @@ const schema = Joi.object().keys({
   'middle-names': Joi.string().allow('').max(100).optional().label('Middle names').meta({ componentType: 'textbox' }),
   'last-name': Joi.string().max(100).label('Last name').meta({ componentType: 'textbox' }),
   'submit': Joi.any().optional().strip()
-}).or('first-name', 'middle-names', 'last-name');
+}).or('name-title', 'first-name', 'middle-names', 'last-name');
```

Comment:
this will mean that only 1 of the above is required, I think you want to make title .required on it's own?